src/site/twitch.tv/modules/emote-menu/EmoteMenu.vue

```diff
@@ -186,6 +187,14 @@ watch(
 	{ immediate: true },
 );
 
+// in the "Bellow input" format, sometimes the positioning styles for the menu are not correct
+// when first opened.I manually force the position to be recalculated
+watch(ctx, (newVal) => {
+	if (newVal.open) {
+		uiFloatingMiddleware.value = [shift({ mainAxis: true, crossAxis: true })];
```

Comment:
There were some problems with the first launch of the menu window for setting the Bellow input. For some reason, the menu is not correctly positioned on the first click. I decided to add a watcher that monitors the opening and starts recalculating the position of the menu styles. Perhaps not the best solution. Can you tell me the best way to do this?